modules/go-setting-environment-annotations.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * /gitops/health-information-for-resources-deployment.adoc
+
+:_content-type: REFERENCE
+[id='go-setting-environment-annotations_{context}']             
+= Setting up environment labels and annnotations
+
+[discrete]
+== Environment labels:
+
+The environment application manifest should include `openshift.gitops/environment` label, and `destination.namespace` set. The set value for both the environment name and destination namespace should be the same.
+
+.Syntax
+[source,yaml]
+----
+spec:
+  labels:
+    openshift.gitops/environment: <environment-name>
+  destination:
+    namespace: <environment-name>
+----   
+
+.Example:
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: dev-env
+  namespace: openshift-gitops
+spec:
+  labels:
+    openshift.gitops/environment: dev-env
+  destination:
+    namespace: dev-env
+...
+----
+
+[discrete]
+== Environment annotations:
+The environment namespace manifest has to have `annotations.app.openshift.io/vcs-uri` and `annotations.app.openshift.io/vcs-ref` to specify the version controller code source of the application. The namespace name has to match the environment name.
+
+.Syntax
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    app.openshift.io/vcs-uri: <application-source-url>
+    app.openshift.io/vcs-ref: <branch-reference>
+  name: <environment-name>
+...
+----
+
+.Example:
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    app.openshift.io/vcs-uri: https://github.com/your-domain/your-gitops.git
```

Comment:
```suggestion
    app.openshift.io/vcs-uri: https://example.com/your-domain/your-gitops.git
```
- Per the ISG. "If you must use a web address as an example, do not create an address or use an address that links to (or might link to) an actual website. To avoid inadvertently using an address of an actual website, use addresses that are reserved for use in examples." See here for more details:  https://www.ibm.com/docs/en/ibm-style?topic=elements-web-ip-email-addresses#using-addresses-in-examples